src/components/Envelope.vue

```diff
@@ -350,15 +350,19 @@ export default {
 		toggleSelected() {
 			this.$emit('update:selected', !this.selected)
 		},
-		onClick() {
-			if (this.draft) {
-				this.$store.dispatch('showMessageComposer', {
-					data: {
-						...this.data,
-						draftId: this.data.databaseId,
-					},
-					templateMessageId: this.data.databaseId,
-				})
+		onClick(event) {
+			if (event) {
```

Comment:
I don't understand why `event` would be truthy only in some cases. Don't we always get an event object from Vue?

Reply:
i honestly dont know, if i do `onClick(event) {
 event.preventDefault() }`
i doesnt work, if i add the if statement, it works, maybe my approach is wrong

Reply:
Multiselect works now, but you can't open a draft anymore. 

The `event` is never falsy. So `onClick` always goes into the `if` branch, never `else`.